crates/router/tests/connectors/connector_auth.rs

```diff
@@ -48,7 +49,6 @@ pub struct ConnectorAuthentication {
 impl ConnectorAuthentication {
     #[allow(clippy::expect_used)]
     pub(crate) fn new() -> Self {
-        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_auth.toml"`
```

Comment:
Why was this line removed?

Reply:
Was asked to remove in one of the Peer review.